actions/workflows/ngi_uu_workflow.yaml

```diff
@@ -13,26 +13,48 @@ workflows:
             - remote_host
             - trigger_ngi_pipeline
         output:
-            #std_out: <% $.std_out %>
             output_the_whole_workflow_context: <% $ %>
         tasks:
-            #            run_bcl2fastq:
-            #                action: arteria-packs.demultiplex
-            #                input:
-            #                    host: "<% $.host %>"
-            #                    runfolder: "<% $.runfolder %>"
-            #                    run_type: "<% $.run_type %>"
-            #                publish:
-            #                    status_message: "Runfolder <% $.runfolder %> run on <% $.host %>..."
-            #                on-success:
-            #                    - run_sisyphus_quick_report
-            #         - run_sisyphus_local_processing
+            get_runfolder_name:
+                action: core.local
+                input:
+                    cmd: python -c 'import sys; sys.stdout.write("<% $.runfolder %>".split("/")[-1])'
+                publish:
+                    runfolder_name: <% $.get_runfolder_name.stdout %>
+                    report_err: <% $.get_runfolder_name.stdout %> <% $.get_runfolder_name.stderr %>
```

Comment:
This `report_err` and the following under `get_flowcell_name` need to match the variable printed in `print_error` (which is `siswrap_err` in this workflow). Perhaps all three should be changed to something else.